setup.py

```diff
@@ -192,6 +192,11 @@
         requires['core'] + requires['downloaders'] +
         requires['publish'] + requires['metadata'],
     python_requires='>=3.6',
+    project_urls={'Homepage': 'https://www.datalad.org',
+                  'Developer docs': 'http://docs.datalad.org/en/stable',
+                  'User handbook': 'http://handbook.datalad.org',
```

Comment:
```suggestion
                  'Developer docs': 'https://docs.datalad.org/en/stable,
                  'User handbook': 'https://handbook.datalad.org',
```
stay "safe" for all? ;)

Reply:
thanks for the suggestion! I applied it